torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -291,10 +288,8 @@ def __init__(
         edge_type, edge_label_index = get_edge_label_index(
             data, edge_label_index)
         if edge_label is None:
-            edge_label = edge_label_index.new_zeros(edge_label_index.size(1))
-            edge_label = (edge_label +
-                          1 if self.neg_sampling_ratio == 0 else edge_label)
```

Comment:
The documentation says that `If edge_label does not exist, it will be automatically created and represents a binary classification task (1 = edge, 0 = no edge).`. Either update the doc to say what happens when `neg_sampling_ratio=0` and `edge_label=None` or keep this line.

Reply:
Good point. Thanks!